apps/docs/content/components/collections/Listbox.mdx

```diff
@@ -59,6 +59,7 @@ The Section component represents a `section` within a Hopper container.
 #### ListBoxItem
```

Comment:
![image](https://github.com/user-attachments/assets/b6935a20-b5c5-45dd-ac45-38c561dac8ff)

Do we have a design for this? There's room for a bigger Avatar if needed.